ansible/roles/fanout/templates/arista_7060_deploy.j2

```diff
@@ -21,101 +21,33 @@ no spanning-tree vlan {{ device_vlan_range[inventory_hostname] | list | join(','
 aaa authorization exec default local
 aaa root secret 0 {{ lab_admin_pass }}
 !
-username admin privilege 15 role network-admin secret 0 {{ lab_admin_pass }}
+username {{ secret_group_vars["fanout"]["ansible_eos_user"] }} privilege 15 role network-admin secret 0 {{ secret_group_vars["fanout"]["ansible_eos_pass"] }}
```

Comment:
Variable `secret_group_vars` is not available in public repo.

Reply:
Thanks! I reverted this line.